vignettes/ORFikOverview.Rmd

```diff
@@ -139,10 +139,12 @@ footprintsGR
 ```
 
 Now, lets prepare annotations and focus on START and STOP codons.
-```{r eval = TRUE, echo = TRUE, warnings = FALSE} 
+```{r eval = TRUE, echo = TRUE, warnings = FALSE, message=FALSE} 
```

Comment:
I have asked you to check knitr chunk options. It is "warning = FALSE, message = FALSE"